packages/frontend-2/components/project/page/models/StructureItem.vue

```diff
@@ -17,6 +17,9 @@
         <template v-else-if="pendingModel">
```

Comment:
1. Added loading bar when children items are loading
2. Fixed up spacings a bit for children items, where they didn't have any padding and looked broken